backend/module/eCampApi/test/Rest/CampCollaborationTest.php

```diff
@@ -73,7 +107,40 @@ public function testFetchAll() {
         $this->assertEquals(1, $this->getResponseContent()->total_items);
         $this->assertEquals(10, $this->getResponseContent()->page_size);
         $this->assertEquals("http://{$this->host}{$this->apiEndpoint}?page_size=10&userId={$userId}&campId={$campId}&page=1", $this->getResponseContent()->_links->self->href);
-        $this->assertEquals($this->campCollaboration->getId(), $this->getResponseContent()->_embedded->items[0]->id);
+        $this->assertEquals($this->campCollaboration1->getId(), $this->getResponseContent()->_embedded->items[0]->id);
+    }
+
+    public function testFetchByInviteKey() {
+        $inviteKey = $this->campCollaborationInvited->getInviteKey();
+        $this->dispatch("{$this->apiEndpoint}?&inviteKey={$inviteKey}", 'GET');
```

Comment:
```suggestion
        $this->dispatch("{$this->apiEndpoint}?inviteKey={$inviteKey}", 'GET');
```

Reply:
thank you, that slipped through
done with fixup of backend/CampCollaborationService: implement inviteKey as query parameter